Tests/Doctrine/Orm/DateFilterTest.php

```diff
@@ -93,36 +94,129 @@ public function filterProvider()
     {
         return [
             // Properties enabled with valid values
+            // Test after
             [
                 [
                     'properties' => null,
+                    'nullOption' => DateFilter::NULL_EXCLUDED
```

Comment:
missing comma at the end of a line (but run PHP CS Fixer, it will fix it for you).